rumqttc/examples/websocket.rs

```diff
@@ -1,9 +1,11 @@
-use tokio::{task, time};
-
+#[cfg(feature = "websocket")]
```

Comment:
Examples that don't compile due to lack of feature support would fail clippy, hence being feature gated with a panicking alternative.